src/containers/GlossPage/components/ExamplesFieldArray.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { IGlossExample } from '@ndla/types-backend/build/concept-api';
+import { FieldRemoveButton, FieldSection } from '@ndla/forms';
+import { ButtonV2 } from '@ndla/button';
+import { useTranslation } from 'react-i18next';
+import { FieldArray, useFormikContext } from 'formik';
+import LanguageVariantFieldArray from './LanguageVariantFieldArray';
+import FormAccordion from '../../../components/Accordion/FormAccordion';
+
+interface Props {
+  name: string;
+  exampleLists: IGlossExample[][];
+}
+
+const ExamplesFieldArray = ({ name, exampleLists }: Props) => {
+  const { t } = useTranslation();
+
+  const formikContext = useFormikContext<any>();
```

Comment:
Any? Samme spørsmål som under/over

Reply:
Denne blir litt vrien. Siden errors har typen `Record<string, string>` blir det vannskelig å akksessere en verdi som både kan bli definert for både `ConceptFormValues` og `errors` når det er uvisst hvor mange errors det er i et eksempel som indekseres.

Kan hende jeg overser noe lett her, men en mulig fiks blir vel å endre på `validateFormik`-funksjonen til å kunne håndtere arrays 🤔